cmd/kueuectl/app/list/list_clusterqueue.go

```diff
@@ -0,0 +1,252 @@
+		if err != nil {
+			return err
+		}
+		o.Active = activeFlag
+	}
+
+	return nil
+}
+
+func (o *ClusterQueueOptions) Validate() error {
+	if !o.validActiveFlagOptionProvided() {
+		return fmt.Errorf("invalid value for --active flag: %s", o.Active)
+	}
+	return nil
+}
+
+func (o *ClusterQueueOptions) validActiveFlagOptionProvided() bool {
+	return o.Active == string(clusterQueueActiveAll) ||
+		o.Active == string(clusterQueueActiveEmpty) ||
+		o.Active == string(clusterQueueActiveTrue) ||
+		o.Active == string(clusterQueueActiveFalse)
+}
+
+// Run prints the cluster queues.
+func (o *ClusterQueueOptions) Run() error {
+	opts := metav1.ListOptions{LabelSelector: o.LabelSelector, FieldSelector: o.FieldSelector}
+	list, err := o.Client.ClusterQueues().List(o.ctx, opts)
+	if err != nil {
+		return err
+	}
+
+	if len(list.Items) == 0 {
+		return nil
+	}
+
+	o.applyActiveFilter(list)
+
+	return o.PrintObj(list, o.Out)
+}
+
+func (o *ClusterQueueOptions) applyActiveFilter(cql *v1beta1.ClusterQueueList) {
+	if o.Active == string(clusterQueueActiveAll) || o.Active == string(clusterQueueActiveEmpty) {
+		return
+	}
+
+	filtered := make([]v1beta1.ClusterQueue, 0, len(cql.Items))
+	for _, cq := range cql.Items {
+		switch o.Active {
+		case string(clusterQueueActiveTrue):
+			if isActiveStatus(&cq) {
+				filtered = append(filtered, cq)
+			}
+		case string(clusterQueueActiveFalse):
+			if !isActiveStatus(&cq) {
+				filtered = append(filtered, cq)
+			}
+		}
+	}
+	cql.Items = filtered
+}
+
+// printClusterQueueTable is a printer function for ClusterQueueList objects.
+var _ printers.ResourcePrinterFunc = printClusterQueueTable
+
+func printClusterQueueTable(obj runtime.Object, out io.Writer) error {
+	tp := printers.NewTablePrinter(printers.PrintOptions{})
+	a := &metav1.Table{
+		ColumnDefinitions: []metav1.TableColumnDefinition{
+			{Name: "Name", Type: "string", Format: "name"},
+			{Name: "Cohort", Type: "string"},
+			{Name: "Pending Workloads", Type: "integer"},
+			{Name: "Admitted Workloads", Type: "integer"},
+			{Name: "Active", Type: "boolean"},
+			{Name: "Age", Type: "string"},
+		},
+		Rows: toTableRows(obj.(*v1beta1.ClusterQueueList)),
+	}
+	return tp.PrintObj(a, out)
+}
+
+func toTableRows(list *v1beta1.ClusterQueueList) []metav1.TableRow {
+	rows := make([]metav1.TableRow, len(list.Items))
+	for index := range list.Items {
+		rows[index] = toTableRow(&list.Items[index])
+	}
+	return rows
+}
+
+func toTableRow(cq *v1beta1.ClusterQueue) metav1.TableRow {
+	return metav1.TableRow{
+		Object: runtime.RawExtension{Object: cq},
+		Cells: []interface{}{
+			cq.Name,
+			cq.Spec.Cohort,
+			cq.Status.PendingWorkloads,
+			cq.Status.AdmittedWorkloads,
+			isActiveStatus(cq),
+			duration.HumanDuration(time.Since(cq.CreationTimestamp.Time)),
+		},
+	}
+}
+
+func isActiveStatus(cq *v1beta1.ClusterQueue) bool {
+	return meta.IsStatusConditionPresentAndEqual(cq.Status.Conditions, v1beta1.ClusterQueueActive, metav1.ConditionTrue)
```

Comment:
```suggestion
	return meta.IsStatusConditionTrue(cq.Status.Conditions, v1beta1.ClusterQueueActive)
```
 And maybe just call the meta func from filter.